src/mito2/src/sst/file.rs

```diff
@@ -134,6 +134,8 @@ pub struct FileMeta {
     /// the default value `0` doesn't means the file doesn't contains any rows,
     /// but instead means the number of rows is unknown.
     pub num_row_groups: u64,
+    /// Max sequence in this file
+    pub max_sequence: u64,
```

Comment:
idea: Can we use `Option<NonZeroU64>`? So we won't forget that this field can be zero.